Movieee/Movieee/UIs/Movie Detail Screen/MovieDetailViewController.swift

```diff
@@ -15,7 +18,12 @@ final class MovieDetailViewController: UIViewController {
     @IBOutlet private weak var genreCollection: UICollectionView!
     @IBOutlet private weak var castCollection: UICollectionView!
     @IBOutlet private var ratingStars: [UIImageView]!
+    
     var idFilm = 0
+    private var items = [FavoriteModel]()
+    private var movieHasFavorited = true
```

Comment:
```suggestion
    private var isMovieHasFavorited = true
```
Sao lại đặt giá trị default là true vậy e?